src/main/ApplicationImpl.h

```diff
@@ -194,8 +192,10 @@ class ApplicationImpl : public Application
     // is held in the never-committing LedgerTxn in its entirety -- so if it
```

Comment:
This comment could also use an update emphasizing that this exists only to support old tests that commit directly to the LTX and bypass the BL, and should be removed in future sometime (and linking to bug).